package.json

```diff
@@ -120,10 +103,398 @@
+        "./packages/victory-bar:jest",
+        "./packages/victory-box-plot:jest",
+        "./packages/victory-brush-container:jest",
+        "./packages/victory-brush-line:jest",
+        "./packages/victory-candlestick:jest",
+        "./packages/victory-canvas:jest",
+        "./packages/victory-chart:jest",
+        "./packages/victory-core:jest",
+        "./packages/victory-create-container:jest",
+        "./packages/victory-cursor-container:jest",
+        "./packages/victory-errorbar:jest",
+        "./packages/victory-group:jest",
+        "./packages/victory-histogram:jest",
+        "./packages/victory-legend:jest",
+        "./packages/victory-line:jest",
+        "./packages/victory-pie:jest",
+        "./packages/victory-polar-axis:jest",
+        "./packages/victory-scatter:jest",
+        "./packages/victory-selection-container:jest",
+        "./packages/victory-shared-events:jest",
+        "./packages/victory-stack:jest",
+        "./packages/victory-tooltip:jest",
+        "./packages/victory-voronoi:jest",
+        "./packages/victory-voronoi-container:jest",
+        "./packages/victory-zoom-container:jest"
+      ]
+    },
+    "storybook:base": {
+      "dependencies": [
+        "./packages/victory-native:build:lib:esm",
+        "./packages/victory-vendor:build:lib:esm",
+        "./packages/victory:build:lib:esm",
+        "./packages/victory-area:build:lib:esm",
+        "./packages/victory-axis:build:lib:esm",
+        "./packages/victory-bar:build:lib:esm",
+        "./packages/victory-box-plot:build:lib:esm",
+        "./packages/victory-brush-container:build:lib:esm",
+        "./packages/victory-brush-line:build:lib:esm",
+        "./packages/victory-candlestick:build:lib:esm",
+        "./packages/victory-canvas:build:lib:esm",
+        "./packages/victory-chart:build:lib:esm",
+        "./packages/victory-core:build:lib:esm",
+        "./packages/victory-create-container:build:lib:esm",
+        "./packages/victory-cursor-container:build:lib:esm",
+        "./packages/victory-errorbar:build:lib:esm",
+        "./packages/victory-group:build:lib:esm",
+        "./packages/victory-histogram:build:lib:esm",
+        "./packages/victory-legend:build:lib:esm",
+        "./packages/victory-line:build:lib:esm",
+        "./packages/victory-pie:build:lib:esm",
+        "./packages/victory-polar-axis:build:lib:esm",
+        "./packages/victory-scatter:build:lib:esm",
+        "./packages/victory-selection-container:build:lib:esm",
+        "./packages/victory-shared-events:build:lib:esm",
+        "./packages/victory-stack:build:lib:esm",
+        "./packages/victory-tooltip:build:lib:esm",
+        "./packages/victory-voronoi:build:lib:esm",
+        "./packages/victory-voronoi-container:build:lib:esm",
+        "./packages/victory-zoom-container:build:lib:esm"
+      ]
+    },
+    "storybook:build": {
+      "command": "build-storybook",
+      "files": [
+        ".storybook",
+        "stories"
+      ],
```

Comment:
I think we also need `packages/**/*.stories.*` here

Reply:
Oh! Thanks! I'll add that.